src/ScheduledTasks/Models/ScheduleConfiguration.cs

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
TODO: scheduleconfiguration, schedulecreationoptions, scheduleupdateoptions are pojos based on same schema, need to refactor to remove dup codes